Wrappers/Python/ccpi/web_viewer/README.md

```diff
@@ -0,0 +1,10 @@
+To setup your environment for using the web application we recommend using conda as it can simplify things in comparison to other implementations of python environment handling.
+
+Follow these steps:
+- Install a variant of conda for this example we will use mambaforge (includes mamba (faster conda implementation) and has conda-forge added as a default channel), it can be downloaded here: https://github.com/conda-forge/miniforge/releases
+- Create the conda environment, activate then install the required items:
+    - `mamba create -n cilviewer_webapp python==3.9`
```

Comment:
surely we need the viewer too?!

Reply:
This is just for development hence it being in the web_viewer folder

Reply:
But you can't run it without the viewer - we can discuss on monday?

Reply:
Sure :+1:

Reply:
I suppose that you should add an `environment.yaml` file to specify how to install the webapp. See for instance [here](https://stackoverflow.com/a/35245610/1419293) on how to mix conda and pip requirements.